src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BlobProcessorTest.java

```diff
@@ -20,22 +21,39 @@ public void setUp() throws Exception {
     }
 
     @Test
-    public void should_process_zipfile_after_upload_and_set_status() throws Exception {
+    public void should_process_zipfile_after_upload_and_set_status() {
         List<String> files = Arrays.asList("1111006.pdf", "1111002.pdf");
         String metadataFile = "1111006_2.metadata.json";
         String destZipFilename = testHelper.getRandomFilename("24-06-2018-00-00-00.test.zip");
 
+        // valid zip file
+        EnvelopeResponse envelope = uploadZipFile(files, metadataFile, destZipFilename);
+
+        assertThat(ImmutableList.of(Status.NOTIFICATION_SENT, Status.COMPLETED)).contains(envelope.getStatus());
+        assertThat(envelope.getScannableItems()).hasSize(2);
+        assertThat(envelope.getScannableItems()).noneMatch(item -> Strings.isNullOrEmpty(item.documentUuid));
+    }
+
+    @Test
+    public void should_process_zipfile_with_supplementary_evidence_with_oce_classification() {
+        List<String> files = Collections.singletonList("1111006.pdf");
+        String metadataFile = "1111006_3.metadata.json";
```

Comment:
it would be nice to rename those files (on a separate PR maybe)

Reply:
raised #958